agent_client/python/scripts/generate.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -e
+
+
+if ! command -v npx &> /dev/null
+then
+    echo "npx could not be found"
+    exit
+fi
+
+npm list -g | grep @openapitools/openapi-generator-cli || npm install -g @openapitools/openapi-generator-cli
```

Comment:
The command CLI should be callable via the `npx` (https://docs.npmjs.com/cli/v8/commands/npx) even without installation (it fetches the package if it is not installed locally or globally).